xenstore.opam

```diff
@@ -1,35 +1,38 @@
+# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-maintainer: "dave@recoil.org"
+synopsis: "Xenstore protocol in pure OCam"
+description: """
```

Comment:
There used to be a longer description here, see the previous .opam file, can we keep it?

Reply:
Ahh - missed it. Will fix